### PR TITLE
Proposal for additional artsist and related base classes based on needs for PhD course.

### PR DESCRIPTION
**Pull request by tomvanmele**
### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [x] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.

### Checklist

1. [ ] Add the change to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading: `Added`, `Changed`, `Removed`.
1. [ ] Run all tests on your computer (i.e. `invoke test`).
1. [ ] If you add new functions/classes, check that:
   1. [ ] Are available on a second-level import, e.g. `compas.datastructures.Mesh`.
   1. [ ] Add unit tests (especially important for algorithm implementations).

### CODE REVIEW

**Comment by tomvanmele** [comment]
noticed that there is a lot of unintended garbage in this PR. will try to clean up and resubmit...

**Comment by Licini** [comment]
lets get this merged and work from this?

File: src/compas_rhino/artists/pointartist.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+from __future__ import division
+
+import compas_rhino
+from compas_rhino.artists import _PrimitiveArtist
+
+
+__all__ = ['PointArtist']
+
+
+def list_like(target, value, fillvalue=None):
```
Comment (gonzalocasas) [inline src/compas_rhino/artists/pointartist.py:12]: This one feels like an intruder from another PR. ;)

Comment (tomvanmele) [reply src/compas_rhino/artists/pointartist.py:12]: yes i know i needed a temp fix, but will replace it as soon as available :)

**Pull request merged**